pennylane/qnodes/passthru.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+PassthruQNode class
+"""
+import pennylane as qml
+import pennylane.operation
+import pennylane.circuit_graph
+from .base import BaseQNode, QuantumFunctionError
+
+
+
+class PassthruQNode(BaseQNode):
```

Comment:
Shouldn't it actually be `PassthroughQNode`? Or can you use both?

Reply:
It's a less common spelling, but common enough to be found in dictionaries.
https://www.merriam-webster.com/dictionary/thru
I just used it because it's short :)

Reply:
"Thru" is pretty much an abbreviation of "Through", which appears when the word is combined with others (e.g., "Drive-Thru"). It makes sense in this case :)